packages/web3/src/codec/instr-codec.ts

```diff
@@ -0,0 +1,462 @@
+export const IsPaying: Instr = { code: 0xa6, value: {} }
+export const TransferAlph: Instr = { code: 0xa7, value: {} }
+export const TransferAlphFromSelf: Instr = { code: 0xa8, value: {} }
+export const TransferAlphToSelf: Instr = { code: 0xa9, value: {} }
+export const TransferToken: Instr = { code: 0xaa, value: {} }
+export const TransferTokenFromSelf: Instr = { code: 0xab, value: {} }
+export const TransferTokenToSelf: Instr = { code: 0xac, value: {} }
+export const CreateContract: Instr = { code: 0xad, value: {} }
+export const CreateContractWithToken: Instr = { code: 0xae, value: {} }
+export const CopyCreateContract: Instr = { code: 0xaf, value: {} }
+export const DestroySelf: Instr = { code: 0xb0, value: {} }
+export const SelfContractId: Instr = { code: 0xb1, value: {} }
+export const SelfAddress: Instr = { code: 0xb2, value: {} }
+export const CallerContractId: Instr = { code: 0xb3, value: {} }
+export const CallerAddress: Instr = { code: 0xb4, value: {} }
+export const IsCallerFromTxScript: Instr = { code: 0xb5, value: {} }
+export const CallerInitialStateHash: Instr = { code: 0xb6, value: {} }
+export const CallerCodeHash: Instr = { code: 0xb7, value: {} }
+export const ContractInitialStateHash: Instr = { code: 0xb8, value: {} }
+export const ContractInitialCodeHash: Instr = { code: 0xb9, value: {} }
+export const MigrateSimple: Instr = { code: 0xba, value: {} }
+export const MigrateWithFields: Instr = { code: 0xbb, value: {} }
+export const CopyCreateContractWithToken: Instr = { code: 0xbc, value: {} }
+export const BurnToken: Instr = { code: 0xbd, value: {} }
+export const LockApprovedAssets: Instr = { code: 0xbe, value: {} }
+export const CreateSubContract: Instr = { code: 0xbf, value: {} }
+export const CreateSubContractWithToken: Instr = { code: 0xc0, value: {} }
+export const CopyCreateSubContract: Instr = { code: 0xc1, value: {} }
+export const CopyCreateSubContractWithToken: Instr = { code: 0xc2, value: {} }
+export const LoadMutFieldByIndex: Instr = { code: 0xc3, value: {} }
+export const StoreMutFieldByIndex: Instr = { code: 0xc4, value: {} }
+export const ContractExists: Instr = { code: 0xc5, value: {} }
+export const CreateContractAndTransferToken: Instr = { code: 0xc6, value: {} }
+export const CopyCreateContractAndTransferToken: Instr = { code: 0xc7, value: {} }
+export const CreateSubContractAndTransferToken: Instr = { code: 0xc8, value: {} }
+export const CopyCreateSubContractAndTransferToken: Instr = { code: 0xc9, value: {} }
+export const NullContractAddress: Instr = { code: 0xca, value: {} }
+export const SubContractId: Instr = { code: 0xcb, value: {} }
+export const SubContractIdOf: Instr = { code: 0xcc, value: {} }
+export const AlphTokenId: Instr = { code: 0xcd, value: {} }
+export const LoadImmField = (index: number): Instr => ({ code: 0xce, value: { index } })
+export const LoadImmFieldByIndex: Instr = { code: 0xcf, value: {} }
+
+export class InstrCodec implements Codec<Instr> {
```

Comment:
We added some new instructions in the Rhone upgrade. Do we need to support them now?

Reply:
I will add, since they it wouldn't break anything here.